app/webpacker/packs/quota-definition-chart.js

```diff
@@ -26,6 +28,24 @@ const config = {
         fill: true,
         data: newBalances,
       },
+      {
+          label: "Critical threshold",
+          borderColor: "#ff0000",
+          backgroundColor: "rgba(29, 112, 184, 0.1)",
+          fill: false,
+          data: criticalThresholds,
+          // Makes the plotted dots smaller
+          radius: 2
+      },
+      {
+        label: "Initial volume",
+        borderColor: "#000000",
+        backgroundColor: "rgba(29, 112, 184, 0.1)",
+        fill: false,
+        data: initialVolumes,
+        // Makes the plotted dots smaller
+        radius: 2
```

Comment:
Only suggestion would be use a const rather then a comment?

```javascript
const smallerDot = 2;

{
  radius: smallerDot
}
```

Reply:
Love it and hate magic numbers